volumes/miovision/sql/README.md

```diff
@@ -317,6 +318,19 @@ Used to determine the maximum acceptable gap for use in `unacceptable_gaps` tabl
 
 ## Reference Tables
 
+### `miovision_api.discontinuities`
+The discontinuity table contains `Moments in time when data collection methods changed in such a way that we would expect clear pre- and post-change paradigms that may not be intercomparable`.
+
+| column_name | Comments | data_type | sample |
+| ------ | ----------- |
+| uid    | simple incrementing primary key | serial | 1 |
+| intersection_uid | Reference to intersections table. A null intersection_uid refers to all intersections. | integer
+| classification_uid | Reference to classifications table. A null classification_uid refers to all classifications. | integer
+| break | Moment the change takes place | timestamp
+| give_or_take | approximate bounds if the precise time is not known | interval
+| notes | required description of what changed - be verbose! | text
```

Comment:
I don't think this table is formatted properly. It's not rendering right for me.

Reply:
Whoops. Fixed now.